### DISCUSSION TITLE
Ask for help!

### DISCUSSION THREAD

**Discussion by xiongguobing** [getlantern/lantern-forum-cn#122] [Q&A] [answered]
您好！作为一个铁杆lantern用户,最近的事情很糟心,我仔细看了之前记录,发现了几个问题,现提出并请部分退费（退两次）,谢谢！ 一是2023-11-27我已经交了下一年度费用,但2024-1-28提示我会费到期,我立即缴费,但当时链接lantern 后网络提示未成功,我再次操作缴费一年的费用成功了，后面扣款却是两次，我查看账户到期却是2027年了,也就是说,2024-1-28那次费用到期不能连接的提示是有误，且2024-1-28那次缴费的第一次实际也是成功的,你们可以查看我的账户（就是本邮箱地址）及截图（一个是2023-11-27的缴费单据,另一个是2024-1-28两次缴费有记录但一直没有单据的截图）,可以确定,因此我要求将最近两次的缴费退了,保留2023-11-27的缴费、即保留账户到2025。谢谢！ 二是我现在更换为iphone 15,反复尝试不能下载lantern 安装,向你们求助了，也没有解决,再次寻求帮助,能否直接邮箱一个app,谢谢! 三是附件添加图片反复不能成功，何解？ 作为一个铁杆用户,希望你们越来越好！ 祝新年快乐！
我的账户xiongguobing.md@qq.com

**Comment by xiongguobing** [comment]
这个回复根本就没有解决问题？？？！！！

--reply above this line--

你好，

我们已收到您关于 Lantern 应用程序功能问题的反馈。 您的报告表明您的问题与我们列出的任何问题类型都不匹配。 鉴于您的问题的独特性，我们将让我们的一名技术人员审查您的报告，以便为您提供更合适的解决方案。 同时，您可以在我们的常见问题解答页面上找到大量故障排除资源。 感谢您在此过程中的耐心和理解，因为我们是一个小团队，致力于解决大客户的问题。

真挚地，
The Lantern Team

有用的网址！ （如果我们的主链接被阻止，请使用提供的辅助链接）
故障诊断 - https://lantern.io/faq - alternate: (https://www.mror.network/faq)
下载 蓝灯 - https://lantern.io/download - alternate: (https://www.mror.network/download)
故障诊断 - https://lantern.io/faq#trouble - alternate: (https://www.mror.network/faq#trouble)

https://lantern.jitbit.com/helpdesk/User/AutoLogin?username=xiongguobing.md%40qq.com&email=xiongguobing.md%40qq.com&userHash=028a640d97db46836ae0d13f1e45ee9e&ReturnUrl=https%3A%2F%2Flantern.jitbit.com%2Fhelpdesk%2FTicket%2F69067152



您好！作为一个铁杆lantern用户,最近的事情很糟心,我仔细看了之前记录,发现了几个问题,现提出并请部分退费（退两次）,谢谢！ 一是2023-11-27我已经交了下一年度费用,但2024-1-28提示我会费到期,我立即缴费,但当时链接lantern 后网络提示未成功,我再次操作缴费一年的费用成功了，后面扣款却是两次，我查看账户到期却是2027年了,也就是说,2024-1-28那次费用到期不能连接的提示是有误，且2024-1-28那次缴费的第一次实际也是成功的,你们可以查看我的账户（就是本邮箱地址）及截图（一个是2023-11-27的缴费单据,另一个是2024-1-28两次缴费有记录但一直没有单据的截图）,可以确定,因此我要求将最近两次的缴费退了,保留2023-11-27的缴费、即保留账户到2025。谢谢！ 二是我现在更换为iphone 15,反复尝试不能下载lantern 安装,向你们求助了，也没有解决,再次寻求帮助,能否直接邮箱一个app,谢谢! 三是附件添加图片反复不能成功，何解？ 作为一个铁杆用户,希望你们越来越好！ 祝新年快乐！
status: New | priority: High

NOTE: When replying to this email please leave the subject-line intact.

View our FAQs at [lantern.io/faq](https://lantern.io/faq)
 Download the latest version: [Android](https://s3.amazonaws.com/lantern/lantern-installer.apk) | [MacOS](https://s3.amazonaws.com/lantern/lantern-installer.dmg) | [Linux (32-bit)](https://s3.amazonaws.com/lantern/lantern-installer-32-bit.deb) | [Linux (64-bit)](https://s3.amazonaws.com/lantern/lantern-installer-64-bit.deb) | [Windows](https://s3.amazonaws.com/lantern/lantern-installer.exe)
 Check out our Telegram: [Official Bot](https://t.me/lantern_official_bot) | [فارسی](https://t.me/Lanternpro_Iran) | [Русский](https://t.me/lantern_russia) | [العربية](https://t.me/lantern_uae) | [Türkçe](https://telegram.me/lantern_turkey) | [tiếng Việt](https://telegram.me/lantern_vietnam)
 Follow us on Twitter: [فارسی](https://twitter.com/getlantern_fa) | [English](https://twitter.com/getlantern)
 See us on Facebook: [فارسی](https://www.facebook.com/getlanternpersian) | [English](https://www.facebook.com/getlantern)
 Follow our Instagram: [فارسی](https://www.instagram.com/lantern.io_fa/) | [English](https://www.instagram.com/getlantern/)
 Help spread Lantern by becoming a reseller - [Signup now](https://reseller.lantern.io/signup)
 To support lantern, donate [here](https://donorbox.org/lantern-donation)
🚀 Powered by [Jitbit Helpdesk](https://www.jitbit.com/helpdesk/?utm_source=hd_email_notifications&utm_medium=email&utm_campaign=hd_powered_by)

**Comment by MaxMKMax** [comment]
@xiongguobing 您好, 对于最近发生的问题，我们深感抱歉。谢谢您的反馈。
从现在开始，我将帮助您人工处理所有问题。很抱歉，我们的电子邮件是自动电子邮件，因此无法满足您的期望。

请问您有下载最新版本吗? 蓝灯会修复您遇到的问题和进行优化。请您耐心等待。但在这之前需要您帮我一个忙。

步骤一: 您方便让我知道您现在用的是哪个版本吗？打个比方version xxx.xxx . 然后用安卓手机还是萍果手机? 或者哪款的电脑呢？您可以在留言区给我知道.

步骤二: 然后, 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

步骤三: 当您在app提交了问题报告之后, 请您记得回来这里留言您的log/ticket号码

* 如果这个方法行不通. 你直接邮件我这些资料. mike@getlantern.org

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

非常抱歉影响了您的体验感。谢谢您的等待与支持! 祝您新年快乐！

**Reply by xiongguobing** [reply]
感谢回复！！

步骤一: 您方便让我知道您现在用的是哪个版本吗？
iphone15       ios版本 17.3

步骤二: 然后, 您可以帮助我们在应用程序中提交报告您的问题吗？
使用电脑可以登录，并提交问题，但不能上传截图

步骤三: 当您在app提交了问题报告之后, 请您记得回来这里留言您的log/ticket号码
手机未能下载、安装lantern

诉求1：退两次（一年期的）专业版费用，合计RMB 340+340=680元
诉求2：各种办法都想尽了，iphone15   ios 17.3版还是不能下载lantern app，能否麻烦直接将iOS版的lantern app直接寄一个到邮箱？谢谢！

**Reply by MaxMKMax** [reply]
@xiongguobing 您好. 感谢回复！

诉求1：您好. 您是需要退款这两次对吗? 我帮您安排一下. 不好意思迟回复您了! 
诉求2：我们已经进行修复了. 你试试看能从这个链接下载吗?
https://github.com/getlantern/lantern

**Comment by xiongguobing** [comment]
您好！谢谢您的回复。
现在我的账户到期是2027年，麻烦退款两个一年期，保留账户到期2025，请麻烦退款340+340，谢谢！
另，我马上尝试按照您发的链接试一下，有问题随后向你反馈。





------------------ Original ------------------
From: Mike K ***@***.***&gt;
Date: Fri,Feb 9,2024 0:28 AM
To: getlantern/lantern-forum-cn ***@***.***&gt;
Cc: xiongguobing ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [getlantern/lantern-forum-cn] Ask for help! (Discussion #122)





 
@xiongguobing 您好. 感谢回复！
 
诉求1：您好. 您是需要退款这两次对吗? 我帮您安排一下. 不好意思迟回复您了!
 诉求2：我们已经进行修复了. 你试试看能从这个链接下载吗?
 https://github.com/getlantern/lantern

**Comment by xiongguobing** [comment]
你好，下载不了，进入后点击下载刘进入这个页面，再尝试iphone App Store搜索结果却是这个页面，见附图。能否直接将iPhone lantern app打包通过邮件发给我，谢谢！





------------------ Original ------------------
From: Mike K ***@***.***&gt;
Date: Fri,Feb 9,2024 0:28 AM
To: getlantern/lantern-forum-cn ***@***.***&gt;
Cc: xiongguobing ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [getlantern/lantern-forum-cn] Ask for help! (Discussion #122)





 
@xiongguobing 您好. 感谢回复！
 
诉求1：您好. 您是需要退款这两次对吗? 我帮您安排一下. 不好意思迟回复您了!
 诉求2：我们已经进行修复了. 你试试看能从这个链接下载吗?
 https://github.com/getlantern/lantern

**Reply by MaxMKMax** [reply]
@xiongguobing 您好！好的。我这边已经帮你申请退款了。还要求团队向您发送下载应用程序 apk。请稍等一下，明后天就会为您完成此操作。

**Comment by xiongguobing** [comment]
谢谢您！
静候佳音！


值此中国新春之际，祝您及家人新年快乐，万事如意，龙行龘龘，前程朤朤！





------------------ Original ------------------
From: Mike K ***@***.***&gt;
Date: Fri,Feb 9,2024 6:22 PM
To: getlantern/lantern-forum-cn ***@***.***&gt;
Cc: xiongguobing ***@***.***&gt;, Mention ***@***.***&gt;
Subject: Re: [getlantern/lantern-forum-cn] Ask for help! (Discussion #122)





 
@xiongguobing 您好！好的。我这边已经帮你申请退款了。还要求团队向您发送下载应用程序 apk。请稍等一下，明后天就会为您完成此操作。

**Comment by MaxMKMax** [comment, marked as answer]
@xiongguobing 您好！感谢您的祝福! 

1. 您可以发送电子邮件至 downloads@getlantern.org 吗？它会自动回复您并指导您下载应用程序。这是我们建议那些遇到下载问题的用户的最佳方法。

https://lantern.io/zh-Hans/faq#use-2 
这篇文章可能会帮助您解决下载应用程序的问题。您可以慢慢读。谢谢。

2. 退款的事已经在进行了, 应该很快就好了! 

美国蓝灯团队祝福您:
当春风拂过万物，唤醒沉睡的大地，我们迎来了又一个充满希望的新春。在这辞旧迎新的美好时刻，愿诗意的语言带给您和您的家人最深的祝福。愿新的一年，您的生活如诗如画，幸福安康；愿您步步高升，事业蒸蒸日上；愿家庭和睦，笑容常在。春节快乐，万事如意！

谢谢您的支持! 新春快乐!

**Comment by qrj123562** [comment]


**Reply by MaxMKMax** [reply]
@qrj123562 您好. 我马上给您安排一下手动升级. 不好意思让您久等了. 非常抱歉. 谢谢您的支持!

**Reply by MaxMKMax** [reply]
@qrj123562 下午好. 请问您有成功被升级到专业版了吗?
